### DISCUSSION TITLE
Demo发散

### DISCUSSION THREAD

**Discussion by FrankWhh** [PaddlePaddle/PaddleSpeech#653] [Q&A]

**Comment by zh794390558** [comment]
please giving more information.

**Reply by xialei2821212670** [reply]
![image](https://user-images.githubusercontent.com/57654726/126127608-59ca35d1-1fee-4a45-804b-482bd675c2a2.png)
你好，我调整了dem_server,使用aishell预测，服务端接收本地请求。
![image](https://user-images.githubusercontent.com/57654726/126128915-bdec1d67-d5aa-442f-9e8a-93cea30b64eb.png)
使用1.8的代码

**Reply by zh794390558** [reply]
这个应该是输入的语音问题。转成8K试试。建议使用develop的最新代码，1.x不再维护了。
